referenceframe/model_json_test.go

```diff
@@ -27,6 +27,15 @@ func TestParseJSONFile(t *testing.T) {
 		"referenceframe/testjson/worldjoint.json",
 		"referenceframe/testjson/worldlink.json",
 		"referenceframe/testjson/worldDH.json",
+		"referenceframe/testjson/missinglink.json",
+	}
+
+	badFilesErrorMsgs := []string{
```

Comment:
I like the improvement to this test! These should all use the functions or variables outlined in `referenceframe/errors.go` though.